code/formatters.py

```diff
@@ -12,6 +12,11 @@
     ","
 )
 
+DEFAULT_SEPARATOR = ' '
+SMASH_SEPARATOR = ''
+SEP = True
```

Comment:
While we're cleaning things up, SEP and NOSEP could use a comment explaining what they mean/do.

Reply:
https://github.com/knausj85/knausj_talon/pull/746